src/assets/js/design-redhat.js

```diff
@@ -1,12 +1,90 @@
 (function($){
 
+/* ============================== */
+/*  MENU
+/* ============================== */
+
+$(document).ready(function(){
```

Comment:
Document ready already exists in the code below. Rather than having a duplicate instance, I'd move your code inside of the existing one below. `.on('ready')` is also a more recommended way of writing quite a few methods now, instead of `.ready()` which was deprecated in jQuery v3.0.

Examples:
`.on('click')`
`.on('mouseover')`
`.on('scroll)`
`.on('ready')`